### DISCUSSION TITLE
Allowing users to choose time ranges for each day

### DISCUSSION THREAD

**Discussion by kleenkanteen** [lukevella/rallly#753] [Ideas]
I wanted to make a site that is when2meet.com, a site I used a lot in uni and nonprofit groups. I wanted a updated UI and a few functional features. rally.co already exists so I am interested in adding my feature to it.

I used whe2meet.com in 5 different groups having 8-20 people. For all except 1 group, the goal was to find a regular weekly time to meet. We were given a few days to select from, from say 9-5pm. People then clicked and dragged their free times like when2meet.com. The time with most overlap was chosen. Here is a picture of a when2meet event:

![image](https://github.com/lukevella/rallly/assets/67878058/1ae66c47-4ef9-4d49-92d7-16178bf4ca81)

On the left is where I can click and drag the specific times I am available on 3 separate days. The organizer said to choose times from 9-5pm. On the right shows the availability overlap of the participants.

Currently rally.co only lets you choose yes/no to a given time range. Someone cannot specify they are only available for half of say, wednesday's time range. Participants should be able to give their specific timings inside the possible times.

How will rally.co incorporate this? In the poll, these checkboxes will change to time ranges like when2meet.com:

![image](https://github.com/lukevella/rallly/assets/67878058/33748ff4-2177-406c-8188-20632962b12a)

If you agree I can take on implementing this feature.

**Comment by lukevella** [comment]
Hi @kleenkanteen, this is something I plan on implementing myself at some point. I think this is too big of a task for a first-time contributor. You're welcome to play with the idea for your own benefit and I would welcome any learnings you get from that but I won't be able to spend the necessary time to review the work and get this merged.

**Reply by kleenkanteen** [reply]
Ok, I will work on some smaller existing issues then before I try this one
